.travis.yml

```diff
@@ -40,6 +40,10 @@ cache:
 services: mongodb
 
 before_install:
+    - fold_start () { echo "travis_fold:start:$(echo $1 | tr / .)"; echo -e "\\e[1;34m$1\\e[0m"; }
+    - fold_end () { echo "travis_fold:end:$(echo $1 | tr / .)"; }
+    - fold_wrap () { $T=$1; shift; fold_wrap_start $T; sh -c "$*" && fold_wrap_end $T; }
```

Comment:
are you sure about `fold_wrap_start` and `fold_wrap_end` ? shouldn't it be `fold_start` and `fold_end` instead ?

And `fold_wrap_end` will not be called if the command is failing. Is it expected ?